cmd/antrea-agent/options.go

```diff
@@ -135,7 +136,12 @@ func (o *Options) validate(args []string) error {
 	}
 
 	if encapMode.SupportsNoEncap() {
-		if !features.DefaultFeatureGate.Enabled(features.AntreaProxy) {
+		// When use noEnCap traffic mode without AntreaProxy, pod-to-service traffic is handled by iptables/ipvs in
```

Comment:
we capitalize K8s terms (e.g. Pod, Service, ...) by convention in the Antrea code base

Reply:
Okay, the capitalization problem has been corrected.